packages/lab/src/checkbox/Checkbox.tsx

```diff
@@ -132,7 +132,8 @@ export const Checkbox = forwardRef<HTMLLabelElement, CheckboxProps>(
         {...rest}
       >
         <input
-          aria-checked={indeterminate ? "mixed" : checked}
+          // aria-checked only needed when indeterminate since native indeterminate behaviour is not used
+          aria-checked={indeterminate ? "mixed" : undefined}
```

Comment:
Could add a test for this just to prevent breaking this behaviour in some future refactor.

Reply:
Test has been added